### ISSUE TITLE
Less than sign "<" turns to unicode with gron tool

### ISSUE THREAD

**Issue by davgibbs** [tomnomnom/gron#22]
Hi @tomnomnom 

Only just started to use the gron tool, so maybe I am missing something!

When I include a "<" as part of my json file, the "groned" data output includes the unicode (http://unicode-table.com/en/#003C) version of the character "<" (\u003c).

```bash
[david.gibbons@7V0DZY1 ~]$ gron --version
gron version 0.3.6
[david.gibbons@7V0DZY1 ~]$ 
[david.gibbons@7V0DZY1 ~]$ cat example.json 
{
	"glossary": {
		"title": "example glossary",
		"GlossDiv": {
			"title": "<S"
		}
	}
}
[david.gibbons@7V0DZY1 ~]$ gron example.json
json = {};
json.glossary = {};
json.glossary.GlossDiv = {};
json.glossary.GlossDiv.title = "\u003cS";
json.glossary.title = "example glossary";
[david.gibbons@7V0DZY1 ~]$ gron example.json | grep '<'
[david.gibbons@7V0DZY1 ~]$
[david.gibbons@7V0DZY1 ~]$ cat example.json | grep '<'
			"title": "<S"
[david.gibbons@7V0DZY1 ~]$
```

So then I cannot grep for "<" directly, even though it is in my json file.

Is this something that can be fixed? Thanks

**Comment by tomnomnom** [comment]
@davgibbs ah! That'll be a setting on the JSON decoder. The escapeHTML option from memory, on the encoder used in the quoteString function in token.go (again from memory).

I'm on holiday in the countryside at the moment with only a phone so I'm afraid I can't do an immediate fix. I'll sort it when I get back to civilisation if someone hasn't beaten me to the punch!

The ideal solution would be to implement `quoteString` 'properly' rather than slightly abusing JSON.Marshal, but setting the escapeHTML option would be the fastest to implement.

Thanks for raising the issue!

**Comment by tomnomnom** [comment]
@davgibbs hi! I'm back to civilisation now. Your issue should be fixed in [0.3.7](https://github.com/tomnomnom/gron/releases/tag/v0.3.7) - let me know if you have any problems!

Thanks again!

**Comment by davgibbs** [comment]
Just tried the new release there @tomnomnom . Looks good. Thanks!
